corehq/apps/repeaters/repeater_generators.py

```diff
@@ -40,7 +40,7 @@ def get_payload(self, repeat_record, payload_doc):
     def get_headers(self):
         return {'Content-type': self.content_type}
 
-    def get_test_payload(self):
+    def get_test_payload(self, domain=None):
```

Comment:
Sure but if even one requires it then they should all require it to be safe surely?